load-image-k8s.bash

```diff
@@ -3,20 +3,14 @@
 set -eux
 
 NAME="datacater"
-VERSION="alpha.15"
+VERSION="local.13"
```

Comment:
Would it make sense to generate a unique image version, e.g., using the current timestamp, to avoid having to update this line with each change?

Consider it as food for thought, does not have to be implemented in this PR :)